WhisperAPI/WhisperAPI/Services/SelectSuggestion/SelectSuggestionService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Linq;
+using WhisperAPI.Models;
+
+namespace WhisperAPI.Services.SelectSuggestion
+{
+    public class SelectSuggestionService : ISelectSuggestionService
+    {
+        public bool UpdateContextWithSelectedSuggestion(ConversationContext conversationContext, SearchQuery searchQuery)
+        {
+            Guid suggestionId = new Guid(searchQuery.Query);
```

Comment:
I think you meant to receive a guid directly, you don't need SearchQuery, this doesn't seem like the right class to use.